src/providers/ethereum/EthereumERC20Provider.js

```diff
@@ -0,0 +1,76 @@
+import Provider from '../../Provider'
+import { padHexStart } from '../../crypto'
+import { ensureAddressStandardFormat, ensureHexEthFormat } from './EthereumUtil'
+
+const sleep = ms => new Promise(resolve => setTimeout(resolve, ms))
```

Comment:
We are using this code very frequently in our codebase. Can we move it to a utility file?